pages/download.md

```diff
@@ -15,6 +15,11 @@ We recommend **[Etcher](https://www.balena.io/etcher/)** to write our images to
   * **[![treehouses](https://avatars1.githubusercontent.com/u/33208073?size=25)treehouses-122.img.gz](http://download.treehouses.io/treehouse-122.img.gz)**
 * **[![treehouses](https://avatars1.githubusercontent.com/u/33208073?size=25)Past Versions](http://download.treehouses.io/)**
 * **[![treehouses](https://avatars1.githubusercontent.com/u/33208073?size=25)Experimental Versions](http://dev.ole.org/)**
+
+
+To do a checksum of the downloaded img.gz, run `shasum "/path/to/img.gz"` in your terminal.
```

Comment:
@jiaxuan-2 can you transform this into a `Note`

Reply:
... forgot notes in mdwiki are special

Reply:
Do I resolve the conflict and do I still need to convert this to a note?

Reply:
@jiaxuan-2 I resolved the conflict for you

Reply:
Thank you.